### ISSUE TITLE
missing chord minor-7

### ISSUE THREAD

**Issue by magicjuho** [trubadur-app/trubadur#1]
missing chord Am7

**Comment by trubadur-app** [comment]
To be filled in by the first person who has the time:

```
Chord: minor-7

Guitar instructions:
Am7: 	[]
A#m7: 	[]
Bm7: 	[]
Cm7: 	[]
C#m7: 	[]
Dm7: 	[]
D#m7: 	[]
Em7: 	[]
Fm7: 	[]
F#m7:	[]
Gm7: 	[]
G#m7: 	[]
```

**Comment by scally1017** [comment]
Chord: minor-7

Guitar instructions:
Am7: 	[5 7 5 5 8 5] , [X 0 2 0 1 0]
A#m7: 	[6 8 6 6 9 6] , [X 1 3 1 2 1]
Bm7: 	[7 9 7 7 10 7] , [X 2 4 2 3 2]
Cm7: 	[8 10 8 8 11 8] , [X 3 5 3 4 3]
C#m7: 	[9 11 9 9 12 9] , [X 4 6 4 5 4]
Dm7: 	[10 12 10 10 13 10] , [X 5 7 5 6 5]
D#m7: 	[11 13 11 11 14 11] , [X 6 8 6 7 6]
Em7: 	[0 2 0 0 3 0] , [X 7 9 7 8 7]
Fm7: 	[1 3 1 1 4 1] , [X 8 10 8 9 8]
F#m7:	[2 4 2 2 5 2] , [X 9 11 9 10 9]
Gm7: 	[3 5 3 3 6 3] , [X 10 12 10 11 10]
G#m7: 	[4 6 4 4 7 4] , [X 11 13 11 12 11]

**Comment by trubadur-app** [comment]
Thanks for you help @scally1017 . This is now included in version 1.0.13.
